### DISCUSSION TITLE
Custom typing indicators with animations

### DISCUSSION THREAD

**Discussion by TJMusiitwa** [flyerhq/flutter_chat_ui#683] [Q&A] [answered]
@demchenkoalex I am just wondering if in v1 or will be implemented in v2, if there is a way to customise the typing indicators from the animated circles. 
For say a holiday season or something, I was thinking of having something like a snowman icon or something in place of the circles. 

I would like to maintain the animated aspect and just replace the circles. 
Happy to hear any thoughts or ideas. Thanks.

**Comment by demchenkoalex** [comment, marked as answer]
not in `v1` for sure cause I am not working on it, but I will attach your request to the `v2` reimplementation https://github.com/flyerhq/flutter_chat_ui/discussions/191

**Reply by TJMusiitwa** [reply]
Alright. Cool thanks.
